controllers/wireguard_controller.go

```diff
@@ -739,6 +749,17 @@ func (r *WireguardReconciler) deploymentForWireguard(m *vpnv1alpha1.Wireguard, i
 									MountPath: "/tmp/wireguard/",
 								}},
 						}},
+					InitContainers: []corev1.Container{
+						{
+							SecurityContext: &corev1.SecurityContext{
+								Privileged:  &privileged,
```

Comment:
lets make this optional as some devs don't have access to run privileged containers and ip_forward is mostly enabled everywhere

Reply:
can be triggered using something like:
```
apiVersion: vpn.example.com/v1alpha1
kind: Wireguard
metadata:
  name: "my-cool-vpn"
spec:
  enable_ip_forwrd_on_init:  True
```

Reply:
I don't have strong opinions here; so let me know your thoughts :)

Reply:
Yeah that would work.